Content.Client/Atmos/UI/GasMixerWindow.xaml.cs

```diff
@@ -74,10 +74,10 @@ public void SetOutputPressure(float pressure)
 
         public void SetNodePercentages(float nodeOne)
         {
-            nodeOne *= 100.0f;
+            nodeOne = (float) Math.Round(nodeOne * 100.0f, 2);
             MixerNodeOneInput.Text = nodeOne.ToString(CultureInfo.InvariantCulture);
 
-            float nodeTwo = 100.0f - nodeOne;
+            float nodeTwo = (float) Math.Round(100.0f - nodeOne, 2);
             MixerNodeTwoInput.Text = nodeTwo.ToString(CultureInfo.InvariantCulture);
         }
```

Comment:
This should use the actual[ string formatting](https://learn.microsoft.com/en-us/dotnet/standard/base-types/standard-numeric-format-strings#fixed-point-format-specifier-f) instead of just rounding the float. See